README.md

```diff
@@ -582,13 +582,34 @@ Subscription subscription = new Subscription();
 subscription.setUrl(new URI("https://example.org/webhooks/1"));
 subscription.addTopic(Subscription.Topic.USER_CREATED);
 subscription.addTopic(Subscription.Topic.USER_TAG_CREATED);
-subscription.addTopic(Subscription.Topic.COMPANY);
-subscription.setAppID("pi3243fa");
 Subscription.create(subscription);
 
+// create a subscribtion and subscribe to events
+Subscription subscription = new Subscription();
+subscription.addTopic(Subscription.Topic.EVENT_CREATED);
+Map<String,ArrayList<String>> metadata = new HashMap<String, ArrayList<String>>();
+ArrayList<String> events = new ArrayList<String>(Arrays.asList("cart"));
+metadata.put("event_names", events);
+subscription.setMetadata(metadata);
+subscription.setMetadata(metadata);
+Subscription.create(subscription);
+
+// update a subscription
+Subscription subscription = Subscription.find("nsub_60ca7690-4020-11e4-b789-4961958e51bd");
+subscription.addTopic(Subscription.Topic.COMPANY_CREATED);
+Subscription updatedSubscription = Subscription.update(subscription);
+
+// delete a subscription
+Subscription subscription = new Subscription();
+subscription.setId("nsub_83793feb-8394-4cb6-91d6-68ef4dd08a8e");
+Subscription deletedSubscription = Subscription.delete(subscription);
+
 // find a subscription
 subscription = Subscription.find("nsub_60ca7690-4020-11e4-b789-4961958e51bd");
 
+// ping a subscription
+Subscription.ping("nsub_60ca7690-4020-11e4-b789-4961958e51bd");
```

Comment:
Might be worth passing in a subscription object here rather than the ID itself. e.g
Subscription.delete(subscription) or
Subscription.update(subscription)
That way they could just use the return object after creating it

Reply:
Added support for Subscription.ping(subscription) 👍